flytekit/clis/sdk_in_container/register.py

```diff
@@ -142,6 +149,13 @@ def register(
     md5_bytes, native_url = remote._upload_file(pathlib.Path(zip_file))
     cli_logger.debug(f"Uploaded zip {zip_file} to {native_url}")
 
+    # Clean-up temporary directory that was used for fast registration
```

Comment:
Can we delete this section?  i like the idea of keeping it around, just in case it needs to be inspected.

Reply:
Yeah sure, not sure how the OS will clean it up then, is that fine?

Reply:
yeah it's the OS's problem